migration/src/m20220101_000005_create_table.rs

```diff
@@ -0,0 +1,23 @@
+use sea_schema::migration::{
```

Comment:
what is this migration doing?

Reply:
it is a dummy migration for making other migrations, sea_orm documentation recommends using it and copying it as it is made from `sea-orm-cli migrate init` however it can be deleted if needed

Reply:
cool - no it's fine to stay